test/services/helpdesk/index.test.js

```diff
@@ -29,11 +33,90 @@ describe('helpdesk service', function() {
 		helpdeskService.remove(testProblem)
 			.then(result => {
 				done();
+			}).catch((error) => {
+				logger.info('Could not remove: ' + error);
+				done();
 			});
 	});
 
 	it('registered the helpdesk service', () => {
 		assert.ok(helpdeskService);
 	});
+
+	it('POST /helpdesk with valid data', () => {
+
+		let postBody = {
+			subject: 'Dies ist ein Titel 2',
+			currentTarget: 'Dies ist der CurrentState',
+			targetState: 'Dies ist der TargetState',
+			category: 'dashboard',
+			schoolId: '5836bb5664582c35df3bc000'
+		};
+
+		return helpdeskService.create(postBody, { payload: {userId: '0000d213816abba584714c0a'}})
+			.then(result => {
+				expect(result.subject).to.equal('Dies ist ein Titel 2');
+			}
+		);
+	});
+
+	it('POST /helpdesk with invalid data', () => {
+
+		let postBody = {
+			subject: 'Dies ist ein Titel 3',
+			currentTarget: 'Dies ist der CurrentState 2',
+			targetState: 'Dies ist der TargetState 2',
+			category: 'dashboard'
+		};
+
+		helpdeskService.create(postBody, { payload: {userId: '0000d213816abba584714c0a'}})
+			.catch(err => {
+				expect(err).to.not.be.undefined;
+				expect(err.code).to.equal(400);
+			}
+		);
+	});
+
+	it('POST /helpdesk with invalid data regarding schoolID', () => {
+
+		let postBody = {
+			subject: 'Dies ist ein Titel 4',
+			currentTarget: 'Dies ist der CurrentState 3',
+			targetState: 'Dies ist der TargetState 3',
+			category: 'dashboard',
+			schoolId: '599ec0bb8e4e364ec18ff46c'
+		};
+
+		helpdeskService.create(postBody, { payload: {userId: '0000d213816abba584714c0a'}})
+			.catch(err => {
```

Comment:
gleiches Problem hier ...
er geht tatsächlich in den then Teil, welcher nicht existiert und somit failed der Test auch nicht ...
Euer Test macht also nicht das was hier wünscht.